src/accordionPanel.story.jsx

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
I'd recommend creating one story for all accordion features, `accordion.story.jsx`.

You can add a story for individual panel usage, and another for usage in a group.